users/replicaJunction/replicaJunction.h

```diff
@@ -0,0 +1,118 @@
+#ifndef USERSPACE
```

Comment:
Could you replace the first few lines, eg, these lines: 
```c
#ifndef USERSPACE
#define USERSPACE
```
with just `#pragma one`, and remove the endif at the end?